### DISCUSSION TITLE
Badge Distribution | form closes on 11/05!

### DISCUSSION THREAD

**Discussion by multispark** [microsoft/RAG_Hack#173] [Announcements]
Hello everyone, thank you again for participating in RAGHack!

The team is working hard to distribute badges. Please fill out the form attached to your project submissions (issues) before 11/05!

Thank you.

**Comment by jeremiah-louis** [comment]
Would all the badges be similar participation badges?

**Reply by multispark** [reply]
They are all the same!

**Reply by jeremiah-louis** [reply]
alright

**Comment by abdullah-k18** [comment]
When will the badges be distributed?

**Comment by multispark** [comment]
Hello all, apologies for the delays and thank you for your patience. We are having some technical difficulties with our distribution. We are aiming to have them in before the holidays!

**Comment by multispark** [comment]
Hello all,

In addition to this message, we have also sent out emails. Please submit the following form.

Form link: https://aka.ms/raghack/survey

We aim to distribute all badges by the end of this month. Thank you for your patience!
